ballerina/tests/Config.toml

```diff
@@ -1,3 +1,6 @@
 [ballerina.observe]
 enabled=true
 provider="noop"
+
+[ballerina.log]
```

Comment:
Any reason to specifically enable debug logs here ? If this was for debugging purposes we should remove this IMO.

Reply:
This was added for testing. Will remove it 👍🏼